Make helm-install-nsm will fail if the nsm-system namespace does not already exist

### Context

I'm installing NSM via the `helm-install-nsm` target in a fresh (Azure AKS) k8s cluster that does not already have the `nsm-system` namespace. The command will fail because I have not created that namespace yet, and [helm will not create the namespace for me](https://github.com/helm/helm/issues/5753).

```console
chris@Azure:~/networkservicemesh$ make helm-init
./scripts/helm-init-wrapper.sh
Using Helm 3, skipping 'helm init'...
chris@Azure:~/networkservicemesh$ SPIRE_ENABLED=false INSECURE=true FORWARDING_PLANE=kernel make helm-install-nsm
./scripts/helm-nsm-install.sh --chart nsm \
--container_repo networkservicemesh \
--container_tag master \
--forwarding_plane kernel \
--insecure true \
--networkservice "icmp-responder" \
--enable_prometheus true \
--enable_metric_collection true \
--nsm_namespace nsm-system

Helm version detected: v3.4.0
Performing chart installation ...+ PID=2012
+ set +o xtrace
+ helm install nsm --timeout 5m --atomic --set org=networkservicemesh,tag=master --set forwardingPlane=kernel --set insecure=true --set networkservice=icmp-responder --set prometheus=true --set metricsCollectorEnabled=true --set global.JaegerTracing=true --set spire.enabled=false,spire.org=networkservicemesh,spire.tag=master --set admission-webhook.org=networkservicemesh,admission-webhook.tag=master --set prefix-service.org=networkservicemesh,prefix-service.tag=master --namespace nsm-system ./deployments/helm/nsm
.....Error: create: failed to create: namespaces "nsm-system" not found
.Done
make: *** [.mk/helm.mk:50: helm-install-nsm] Error 1
```

### Solution

To fix this problem, I ran

```console
$ kubectl create namespace nsm-system
```

Then I re-ran `make helm-install-nsm` and everything worked as expected.

I'd be happy to contribute a PR to fix this issue, the fix is small.

--- a/scripts/helm-nsm-install.sh
+++ b/scripts/helm-nsm-install.sh
@@ -152,7 +152,7 @@ fi
 
 if (! kubectl get namespace | grep -q "$NSM_NAMESPACE" ); then
   echo "Creating namespace: $NSM_NAMESPACE"
-  kubectl create namespace $NSM_NAMESPACE
+  kubectl create namespace "$NSM_NAMESPACE"
 fi
 
 set -o xtrace

--- a/scripts/helm-nsm-install.sh
+++ b/scripts/helm-nsm-install.sh
@@ -150,6 +150,11 @@ else
   exit 1
 fi
 
+if (! kubectl get namespace | grep -q "$NSM_NAMESPACE" ); then
+  echo "Creating namespace: $NSM_NAMESPACE"
+  kubectl create namespace $NSM_NAMESPACE
+fi
+
 set -o xtrace
 # shellcheck disable=SC2086
 $HELM install $VERSION_SPECIFIC_OPTS \

